### PR TITLE
refactor(store,world): simplify constants

### PR DESCRIPTION
**Pull request by alvrs**
- Remove `BYTES_TO_BITS` constant, simplify constants used for bitshifting

### CODE REVIEW

File: packages/world/gas-report.json [hunk 1]
```diff
@@ -3,31 +3,31 @@
...
     "name": "AccessControl: requireAccess (cold)",
-    "gasUsed": 7221
+    "gasUsed": 7071
   },
   {
     "file": "test/AccessControl.t.sol",
     "test": "testRequireAccess",
     "name": "AccessControl: requireAccess (warm)",
-    "gasUsed": 3224
+    "gasUsed": 3074
```
Comment (alvrs) [inline packages/world/gas-report.json:30]: solc what are you doing 🤦‍♂️

File: packages/store/src/ResourceId.sol [hunk 2]
```diff
@@ -3,15 +3,14 @@ pragma solidity >=0.8.21;
 
 type ResourceId is bytes32;
 
-uint256 constant TYPE_BYTES = 2;
-uint256 constant NAME_BYTES = 32 - TYPE_BYTES;
-uint256 constant BYTES_TO_BITS = 8;
+uint256 constant TYPE_BITS = 2 * 8;
+uint256 constant NAME_BITS = 256 - TYPE_BITS;
```
Comment (holic) [inline packages/store/src/ResourceId.sol:7]:
```suggestion
uint256 constant NAME_BITS = 32 * 8 - TYPE_BITS;
```
tiny bit clearer

Comment (holic) [reply packages/store/src/ResourceId.sol:7]: also totally a nit, feel free to ignore, but I like to put the smallest unit on the left

like when I do time calculations, I usually always aim for
```
1000 * 60 * 60 * 24
```

**Review by alvrs — COMMENTED** [review]
✅

**Review by alvrs — COMMENTED** [review]
✅

**Pull request merged**